scripts/aws/aws_ec2_table.csv

```diff
@@ -0,0 +1,117 @@
+﻿Name,	APIName,Memory,vCPUs,OnEMR,EMRcost
```

Comment:
It's feature that you can use.
When you specify the instance type in the config file, you parse this file to get memory and cpu of this instance type.
I realized it does not make sense to support this at the moment, because you need to specify the resources well: (https://aws.amazon.com/blogs/big-data/best-practices-for-successfully-managing-memory-for-apache-spark-applications-on-amazon-emr/)

So currently we are using: "maximizeResourceAllocation": "true", which sets spark default config for resource allocation to max.

Reply:
It's feature that you can use.
When you specify the instance type in the config file, you parse this file to get memory and cpu of this instance type.
I realized that it does not make sense to support this at the moment: (https://aws.amazon.com/blogs/big-data/best-practices-for-successfully-managing-memory-for-apache-spark-applications-on-amazon-emr/)

So currently I'm using: "maximizeResourceAllocation": "true", which sets spark default config for resource allocation to max.